spec/std/system_spec.cr

```diff
@@ -18,4 +18,12 @@ describe System do
       cpu_count.should eq(shell_cpus)
     end
   end
+
+  describe "login" do
+    it "returns String when found" do
+      shell_login = `echo $USER`.strip
```

Comment:
@RX14 Now that I've got the logic to return `Nil`, I'm not certain how to structure this test case to handle the current CI - travis has no $USER